<!DOCTYPE html>
<html lang="en">
<head>
	<link type="text/css" rel="stylesheet" href="style.css" />
    	<style type="text/css">
    	@import url("style.css");
    	</style>
	
    	<meta charset="utf-8">
	
	<title>Frédéric Fyon | Hybrids</title>
	
	<meta name="description" content="A description of the Recombination Hotspot project I'm involved in.">
	
	<meta name="keyword" content="Frédéric Fyon, frederic fyon hybridization, frederic fyon mating systems, mating system, self-fertilization, selfing, selfer, outcrosser, hybridization, introgression, mutation load">
	
	<meta name="viewport" content="width=device-width, initial-scale=1">
	
	<meta name="author" content="Frédéric Fyon">
</head>

<body>
	<header>		
		<nav>
			<ul>
                <li><a id="nav1" href="index.html#home">Home</a></li>
				<li><a id="nav2" href="index.html#moi">About Me</a></li>
				<li><a id="nav3" href="index.html#research">My Research</a></li>
				<li><a id="nav4" href="index.html#publication">Publications</a></li>
				<li><a id="nav5" href="index.html#contact">Contact Me</a></li>
			</ul>
		</nav>

		<div id="media">
			<ul>
				<li><a href="https://www.researchgate.net/profile/Frederic-Fyon" target="_blank" rel="noopener"><img src="images/RG.jpg" alt="ResearchGate logo"></a></li>
				<li><a href="https://www.linkedin.com/in/fr%C3%A9d%C3%A9ric-fyon-636419198/" target="_blank" rel="noopener"><img src="images/In.jpg" alt="LinkedIn logo"></a></li>
				<li><a href="https://github.com/FredericFyon" target="_blank" rel="noopener"><img src="images/GH.webp" alt="GitHub logo"></a></li>
				<li><a href="https://publons.com/dashboard/summary/" target="_blank" rel="noopener"><img src="images/Pub.webp" alt="Publons logo"></a></li>
			</ul>
		</div>
	</header>

    <img id="bannerProject" src="images/hybrid.jpg" alt="Phylogeny including introgression and speciation.">
	
	<main>
        <article class="Project">
                 
        <h1>Introgression patterns in Hybrid Zones between Species with Different Mating Systems</h1>
		
	<p class="collab" target="_blank">In collaboration with: <a href="https://waldirmbf.github.io/"> Dr. Waldir Berbel-Filho </a> </p>

        <a id="BackToTop" href=#Top><img src="images/UpArr.png" alt="Link to the top of the page in shape of a curved upward arrow."></a> 

        <div id="Top" class="pro1">
        <h2>Table of contents</h2>
        
        <ol id="ToC">
            <li id="li1"><a href=#Paradox>Current Knowledge</a></li>
            <li><a href=#DR>Death and Resuscitation of Recombination Hotspots</a></li>
	    <li><a href=#RL>Rapidly evolving Recombination Landscapes</a></li>
	    <li><a href=#PRDM9>Evolutionary origin of PRDM9</a></li>
        </ol>
        </div>

        <div id="Current" class="pro3">
            <h2>Current knowledge</h2>
        <p>
           	The concept of <strong>speciation<strong> encompasses a diversity of processes critically important in biodiversity build-up. The result of speciation is the 
		divergence of sister lineages. From a common history, these lineages acquire independent evolutionary trajectories and fate. Some will eventually
		go extinct, others will prosper and give birth to new lineages that will also ultimately diverge. This broadly describe the what has been the dynamics of life
		on earth for billions of year.<br/><br/>
		
		Speciation is often not sudden, but rather progressive and incomplete. This opens the door for <strong>hybridization</strong>, the process by which 
		partially independent lineages mix up and produce hybrid lineages. If these hybrids can reproduce with their parental species (back-crossing), it allows
		for these to exchange genes and share evolutionary fate to a certain extent. The patterns of exhanged genes are called <strong>introgression</strong>, and
		are highly significant in evolutionary biology, biodiversity and conservation.<br/><br/>
		
		Hybridization and introgression often happen between closely-related lineages that have only recently diverged. These lineages are often very similar; however,
		hybridization can also take place between lineages that exhibit important differences. For example, hybridization between self-fertilizers (organisms that fertilize
		themselves) and outcrossers (organisms that fertilize other individuals) have been reported in many plants; at least one example is known in animals. Differences
		in mating systems are expected to result in the evolution of different behavioural, physiological, and genetic traits. Looking at hybrids and introgression patterns
		between such species gives us valuable information on this evolution.
		
		One example of such evolution is the different accumulation of <strong>mutation load</strong> expected in outcrossers and selfers. In selfers, high levels of
		homozygosity are expected to reveal and purge highly deleterious, strongly recessive mutations, otherwise hidden to selection in heterozygote outcrossers. As a result,
		outcrossers accumulate a certain mutation load in the form of highly deleterious, strongly recessive mutations maintained at low frequency. Selfers
		suffer, on the other hand, from higher genetic drift and selection interference. Both mean that slightly deleterious, codominant mutations are expected
		to fix more easily in selfers than in outcrossers. In other words, selfers are expected to accumulate mutation load in the form of fixed slightly
		deleterious, codominant mutations.
	</p>
        </div>

        <div id="DR" class="pro3">
            <h2>Death and Resuscitation of Recombination Hotspots</h2>
		<p>
			To better understand the evolutionary dynamics of PRDM9-directed recombination hotspots, we build the first model formally taking into account:
		</p>	
		
		<ul>
			<li><p>allele matches between PRDM9 and target sequences</p></li>
			<li><p>gene conversion at target loci</p></li>
			<li><p>selection for crossing-over</p></li>
			<li><p>mutation at PRDM9 and target sequences</p></li>
		</ul>
		
		<div id="Dyn">
			<div id="DynText">
		<p> The dynamics of the resulting system can be observed on Fig. 2. We deciphered 3 possible scenarios:</p>
			
            	<ol>
			<li><p>When selection for crossing-overs is strong (<em>f > c/2</em>, with <em>f</em> the fertility cost of an absence of crossing-overs
				and <em>c</em> the gene conversion rate), the system converges towards the fixation of one of the <em>hot</em> haplotypes (A1B1 or A2B2: PRDM9
				allele matches target allele). This coincides with previous results:under intense selection for crossing-overs, hotspots are maintained and are never eroded. </p></li>
			<li><p>When selection for crossing-overs is strong (<em>f > c/2</em>) and initial frequencies are within its basin of attraction, the system converges towards
				an internal equilibrium with intermediary frequencies of all 4 haplotypes.Here an equilibrium is found between the opposing forces of selection and conversion:
				hotspots converge to a mild state. This does not fit empirical data of rapidly evolving recombination landscapes.</p></li>
			<li><p>Otherwise, if initial frequencies are sufficiently extreme, we observe that the equilibrium found between selection and conversion is dynamical</p></li>
		</ol>
			</div>
			<figure id=figImgDyn>
			<img id="imgDyn" src="images/Dyn.jpg" alt="Dynamics of the system studied.">
			<figcaption id="capPRDM9">
			"Fig. 2: Dynamics of a 2-locus 2-allele model describing the mechanism of PRDM9."
			</figcaption>
		</figure>
		</div>
		<div id="Cycle">
			<figure id=figImgCycle>
			<img id="imgCycle" src="images/Cycle.jpg" alt="Death and Resuscitation dynamics.">
			<figcaption id="capCycle">
			Fig. 3: Hotspots die by gene conversion, but are resuscitated by selection for crossing-over.
			</figcaption>
		</figure>
			<div id="CycleText">
		<p>
			Fig. 3 gives an idea of what happens in this latter case. We see that <em>hot</em> haplotypes are converted into <em>cold</em> haplotypes (for example, A1B1 into A1B2).
			Then, however, selection for crossing-overs rescue the hotspot, driving the spread of an alternative PRDM9 allele that binds the converted target allele (for example, 
			A2 spreads in response of the conversion of A1B1 into A1B2; now, A2B2 is the dominant, <em>hot</em> haplotype).<br/><br/>
			
			The Red Queen theory is correct in arguing that selection for alternative PRDM9 alleles is responsible for restoring crossing-over levels. However, we show here
			that there is no need for selecting for PRDM9 alleles that bind new sets of targets. Instead, we argue that PRDM9 should change to bind now <em>cold</em>, but previously <em>hot</em>
			targets. Our model is compaible with empirical data on rapidly evolving PRDM9 and recombination landscapes, without requiring constantly finding new targets
			to bind.
		</p>
			</div>
		</div>
			
			In an extension of the model, we look at the case where there are two target loci. Interestingly, we see on Fig. 4 that crossing-over levels at the two targets tend to fluctuate in
			asynchrony: recombination landscapes vary rapidly while overall crossing-over levels remain stable. This is also a pattern that fits empirical knowledge and is
			difficult to explain via the Red Queen theory.
		</p>
		<figure id=figImgAsynch>
			<img id="imgPRDM9" src="images/Asynch.jpg" alt="Asynchronous oscillations of two hotspots.">
			<figcaption id="capAsynch">
			Fig. 4: Asynchronous oscillations of two hotspots.
			</figcaption>
		</figure>
        </div>

        <div id="RL" class="pro3">
            <h2>Rapidly evolving Recombination Landscapes</h2>
                <p>
                    We ran numerical simulations of this model with a varying number of target loci. An example of results is shown on Fig. XX. We see that in stochastic
			simulations, each target locus fluctuates between <em>hot</em> and <em>cold</em> states, while overall mean crossing-over rates are much more stable.
			Interestingly, we see that two simulations with the same sets of loci and with the same initial conditions rapidly diverge, showing at any given time different
			combinations of <em>hot</em> targets. As such, our model is compatible with the observation that closely-related species exhibit very different
			recombination landscape: even though they use overall the same set of targets, the actual combinations of hotspots can be very different. <br/> <br/>
			
		    <figure id=figImgStoch>
			<img id="imgStoch" src="images/Stoch.jpg" alt="Stochastic simulations with 10 targets.">
			<figcaption id="capStoch">
			Fig. 5: Results obtained from stochastic simulations with 10 target loci.
			</figcaption>
		</figure>
			
			PRDM9 is known from empirical data to exhibit a particularly high mutation rate, 10-fold higher than for other genes. Thought to be in part responsible 
			for PRDM9's fast evolutionary changes, the cause of this feature is still unknown. To explore whether our model can account for that too, we build a 3-locus
			extension with: one target locus, one PRDM9 locus and one mutation modifier locus. <br/> <br/>
			
			Mathematical analysis shows that a modifier allele that increases mutation rates of PRDM9 relative to the target is selectively favored.
			Interestingly, this is not necessarily because such an allele increases individuals' fitness; rather, it is due to linkage effects. 
            </p>
		
		</div>
		<div id="PRDM9" class="pro3">
            <h2>Evolutionary origin of PRDM9</h2>
                <p>
                   Ongoing top-secret work, more in the upcoming months.
            </p>
        </div>
	<div class="publiProject">
		<h2>Published Articles</h2>
		<ul>
				<li> 
					<p>
                    <a class="pdf" href="https://www.biorxiv.org/content/10.1101/2022.09.29.510084v2" target="_blank" rel="noopener">Mutation and the Recombination Hotspot Paradox</a> <br/>
                    Francisco Úbeda, Frederic Fyon, Reinhard Bürger (2022) <strong>bioRxiv PrePrint</strong> <br/>
					DOI: 10.1101/2022.09.29.510084</p>
                </li>
			</ul>
		</div>
    </article>
</main>
</body>
</html>
