<!DOCTYPE html>
<html lang="en">
<head>
	<link type="text/css" rel="stylesheet" href="style.css" />
    	<style type="text/css">
    	@import url("style.css");
    	</style>
	
    	<meta charset="utf-8">
	
	<title>Frédéric Fyon | Recombintion Hotspots</title>
	
	<meta name="description" content="A description of the Recombination Hotspot project I'm involved in.">
	
	<meta name="keyword" content="Frédéric Fyon, frederic fyon recombination, frederic fyon recombination hotspots, recombination hotspot paradox, recombination hotspot unparadox, evolution of recombination hotspots, PRDM9">
	
	<meta name="viewport" content="width=device-width, initial-scale=1">
	
	<meta name="author" content="Frédéric Fyon">
</head>

<body>
	<header>		
		<nav>
			<ul>
                <li><a id="nav1" href="index.html#home">Home</a></li>
				<li><a id="nav2" href="index.html#moi">About Me</a></li>
				<li><a id="nav3" href="index.html#research">My Research</a></li>
				<li><a id="nav4" href="index.html#publication">Publications</a></li>
				<li><a id="nav5" href="index.html#contact">Contact Me</a></li>
			</ul>
		</nav>

		<div id="media">
			<ul>
				<li><a href="https://www.researchgate.net/profile/Frederic-Fyon" target="_blank" rel="noopener"><img src="images/RG.jpg" alt="ResearchGate logo"></a></li>
				<li><a href="https://www.linkedin.com/in/fr%C3%A9d%C3%A9ric-fyon-636419198/" target="_blank" rel="noopener"><img src="images/In.jpg" alt="LinkedIn logo"></a></li>
				<li><a href="https://github.com/FredericFyon" target="_blank" rel="noopener"><img src="images/GH.webp" alt="GitHub logo"></a></li>
				<li><a href="https://publons.com/dashboard/summary/" target="_blank" rel="noopener"><img src="images/Pub.webp" alt="Publons logo"></a></li>
			</ul>
		</div>
	</header>

    <img id="bannerProject" src="images/RHim.webp" alt="Head banner of the page, displaying recombining chromosomes.">
	
	<main>
        <article class="Project">
                 
        <h1>Evolution of Recombination Hotspots in Vertebrates</h1>

        <a id="BackToTop" href=#Top><img src="images/UpArr.png" alt="Link to the top of the page in shape of a curved upward arrow."></a> 

        <div id="Top" class="pro1">
        <h2>Table of contents</h2>
        
        <ol id="ToC">
            <li id="li1"><a href=#Paradox>The Recombination Hotspot Paradox</a></li>
            <li><a href=#DR>Death and Resuscitation of Recombination Hotspots</a></li>
	    <li><a href=#RL>Rapidly evolving Recombination Landscapes</a></li>
	    <li><a href=#PRDM9>Evolutionary origin of PRDM9</a></li>
        </ol>
        </div>

        <div id="Paradox" class="pro3">
            <h2>The Recombination Hotspot Paradox</h2>
        <p>
           	In many species, recombination does not occur uniformly throughout the genome; rather, it usually concentrates in areas of high crossing-over
		activity called Recombination Hotspots. There are two kinds of such hotspots. In some taxa like birds, recombination concentrates near Transcription
		Start Sites. In these cases, the mechanism of recombination localization is thought to be passive: crossing-over happens in these areas as the protein
		machinery takes advantage of open chromatin state associated with the initiation of transcription. In other taxa such as the vast majority of mammals,
		recombination localization is actively determined by a protein called PRDM9. This protein, whose structure has now been determined, uses its highly variable
		zinc-finger subdomains to recognize and bind specific target sequences in the genome. Once bound, PRDM9 facilitates the opening of chromatin and recruits
		the crossing-over machinery.
	</p>
		<figure id=figImgPRDM9>
			<img id="imgPRDM9" src="images/PRDM9.png" alt="Structure of a PRDM9 protein.">
			<figcaption id="capPRDM9">
			"Fig. 1: Typical structure of a PRDM9 protein."
			</figcaption>
		</figure>
	<p>
		Interestingly, this recombination-determination mechanism means that recombination hotspots are self-destructive in nature. Indeed, crossing-over can result
		in the biased gene conversion of the target allele bound by PRDM9 into the homologous allele. As PRDM9 is sequence-specific, heterozygosity means that
		individual hotspots are expected to be progressively eroded, and eventually disappear. Yet, PRDM9-directed hotspots are still alive and going strong in a wide
		range of vertebrate taxa. This discrepancy between prediction and empirical data is what has been referred to as the <strong>Recombination Hotspot Paradox </strong>.
		
		Current resolution of the paradox relies on selection for crossing-overs. Indeed, it has been shown that a reduction in the number of crossing-overs
		can lead to aneuploidy: crossing-overs seem to be involved in the stabilization of the pairing of homologous chromosomes in Prophase I, allowing for their
		correct segregation. If selection for crossing-over is very strong, models predict that it should act against biaised gene conversion and maintain hotspots.
		However, this seems to require unrealistic selection intensity, and is in contradiction with empirical data that show that PRDM9, hotspot localization and overall
		recombination landscapes are all rapidly evolving. If selection is mild, the "Red Queen Theory of Recombination Hotspots" predicts that while individual
		hotspots indeed "die" from biased gene conversion, PRDM9 sequence evolves to "give birth" to new recombination hotspots at alternative target sites.
		
		It is important to note that current resolution of the paradox critically relies on the availability of alternative PRDM9 sequences able to bind new sets of
		targets. This availability has often been supposed, though it may be not that clear. Indeed, suitable target sequences must: (i) be present in sufficient
		numbers in the genome, (ii) exist all throughout the genome, in every chromosome, and (iii) not be present in unsuitable locations for crossing-over,
		such as exons or regulatory sequences. In finite genomes, chances are that the number of alternative target sequence is limited. Arguably,
		the evolutionary process suggested by the Red Queen model would rapidly exhaust them. In some way, it seems that the Red Queen does not resolve the paradox,
		but merely push back the necessary resolution.
        </div>

        <div id="DR" class="pro3">
            <h2>Death and Resuscitation of Recombination Hotspots</h2>
		<p>
			To better understand the evolutionary dynamics of PRDM9-directed recombination hotspots, we build the first model formally taking into account:
		</p>	
		
		<ul>
			<li><p>allele matches between PRDM9 and target sequences</p></li>
			<li><p>gene conversion at target loci</p></li>
			<li><p>selection for crossing-over</p></li>
			<li><p>mutation at PRDM9 and target sequences</p></li>
		</ul>
		
		<div id="Dyn">
			<div id="DynText">
		<p> The dynamics of the resulting system can be observed on Fig. 2. We deciphered 3 possible scenarios:</p>
			
            	<ol>
			<li><p>When selection for crossing-overs is strong (<em>f > c/2</em>, with <em>f</em> the fertility cost of an absence of crossing-overs
				and <em>c</em> the gene conversion rate), the system converges towards the fixation of one of the <em>hot</em> haplotypes (A1B1 or A2B2: PRDM9
				allele matches target allele). This coincides with previous results:under intense selection for crossing-overs, hotspots are maintained and are never eroded. </p></li>
			<li><p>When selection for crossing-overs is strong (<em>f > c/2</em>) and initial frequencies are within its basin of attraction, the system converges towards
				an internal equilibrium with intermediary frequencies of all 4 haplotypes.Here an equilibrium is found between the opposing forces of selection and conversion:
				hotspots converge to a mild state. This does not fit empirical data of rapidly evolving recombination landscapes.</p></li>
			<li><p>Otherwise, if initial frequencies are sufficiently extreme, we observe that the equilibrium found between selection and conversion is dynamical</p></li>
		</ol>
			</div>
			<figure id=figImgDyn>
			<img id="imgDyn" src="images/Dyn.jpg" alt="Dynamics of the system studied.">
			<figcaption id="capPRDM9">
			"Fig. 2: Dynamics of a 2-locus 2-allele model describing the mechanism of PRDM9."
			</figcaption>
		</figure>
		</div>
		
		<p>
			Fig XX gives an idea of what happens in this latter case. We see that <em>hot</em> haplotypes are converted into <em>cold</em> haplotypes (for example, A1B1 into A1B2).
			Then, however, selection for crossing-overs rescue the hotspot, driving the spread of an alternative PRDM9 allele that binds the converted target allele (for example, 
			A2 spreads in response of the conversion of A1B1 into A1B2; now, A2B2 is the dominant, <em>hot</em> haplotype).<br/><br/>
			
			The Red Queen theory is correct in arguing that selection for alternative PRDM9 alleles is responsible for restoring crossing-over levels. However, we show here
			that there is no need for selecting for PRDM9 alleles that bind new sets of targets. Instead, we argue that PRDM9 should change to bind now <em>cold</em>, but previously <em>hot</em>
			targets. Our model is compaible with empirical data on rapidly evolving PRDM9 and recombination landscapes, without requiring constantly finding new targets
			to bind.<br/><br/>
			
			In an extension of the model, we look at the case where there are two target loci. Interestingly, we see on Fig. XX that crossing-over levels at the two targets tend to fluctuate in
			asynchrony: recombination landscapes vary rapidly while overall crossing-over levels remain stable. This is also a pattern that fits empirical knowledge and is
			difficult to explain via the Red Queen theory.
		</p>
        </div>

        <div id="RL" class="pro3">
            <h2>Rapidly evolving Recombination Landscapes</h2>
                <p>
                    We ran numerical simulations of this model with a varying number of target loci. An example of results is shown on Fig. XX. We see that in stochastic
			simulations, each target locus fluctuates between <em>hot</em> and <em>cold</em> states, while overall mean crossing-over rates are much more stable.
			Interestingly, we see that two simulations with the same sets of loci and with the same initial conditions rapidly diverge, showing at any given time different
			combinations of <em>hot</em> targets. As such, our model is compatible with the observation that closely-related species exhibit very different
			recombination landscape: even though they use overall the same set of targets, the actual combinations of hotspots can be very different. <br/> <br/>
			
		    PRDM9 is known from empirical data to exhibit a particularly high mutation rate, 10-fold higher than for other genes. Thought to be in part responsible 
			for PRDM9's fast evolutionary changes, the cause of this feature is still unknown. To explore whether our model can account for that too, we build a 3-locus
			extension with: one target locus, one PRDM9 locus and one mutation modifier locus.
			
			Mathematical analysis shows that a modifier allele that increases mutation rates of PRDM9 relative to the target is selectively favored.
			Interestingly, this is not necessarily because such an allele increases individuals' fitness; rather, it is due to linkage effects. 
            </p>
		
		</div>
		<div id="PRDM9" class="pro3">
            <h2>Evolutionary origin of PRDM9</h2>
                <p>
                   Ongoing top-secret work, more in the upcoming months.
            </p>
        </div>
    </article>
</main>
</body>
</html>
